src/autolabel/models/palm.py

```diff
@@ -120,17 +135,21 @@ def _label(self, prompts: List[str]) -> RefuelLLMResult:
             prompts = [[HumanMessage(content=prompt)] for prompt in prompts]
 
         try:
+            start_time = time()
             result = self._label_with_retry(prompts)
+            end_time = time()
```

Comment:
want to make sure we want to record time including retries

Reply:
Good catch, We want to keep the latency independent of the retry, so should only include the time for a successful call.